docs/faq-server.md

```diff
@@ -3,28 +3,24 @@
 The [SAFE template](safe-template) comes with three alternative server technologies out of the box:
 
 * **Saturn** - A simple, flexible F# abstraction layer that runs on top of Giraffe that enables both MVC- and Web API-style services. 
-* **Giraffe** - A flexible F# framework for creating web-enabled applications. Giraffe runs on top of ASP .NET Core and the Kestrel server. Giraffe's programming model is itself based on Suave's programming model, but has some differences.
-* **Suave** - A flexible F# framework for creating web-enabled applications. Suave comes with its own built-in web server.
-
-## Comparing servers
-
-The diagram below illustrates some of the differences and similarities of the three stacks.
+* **Giraffe** - A flexible F# framework for creating web-enabled applications. Giraffe runs on top of ASP .NET Core and the Kestrel server.
 
 ![](img/safe-server-1.png)
 
- This is also shown in the table below. Both Giraffe's HTTP Handler and  and Suave's WebPart are very similar abstractions; this allows a simple migration path between either server. Meanwhile Saturn, which lives "on top" of Giraffe, provides a set of new higher-level abstractions which delegate down to Giraffe's HTTP Handler.
+Saturn, which lives "on top" of Giraffe, provides a set of new higher-level abstractions which delegate down to Giraffe's HTTP Handler.
 
-| | Saturn | Giraffe | Suave |
-|-|:-:|:-:|:-:|
-| ASP .NET compatible? | Yes | Yes | No |
-| F#-first? | Yes | Yes | Yes |
-| Core Abstractions | scope, controller and HttpHandler | HttpHandler | WebPart
+| | Saturn | Giraffe |
```

Comment:
I'm not sure if this table is needed at all - image and above descriptions explains it well enough

Reply:
Removed